### DISCUSSION TITLE
Code for visualizing complex networks with underlying hyperbolic geometry

### DISCUSSION THREAD

**Discussion by prokolyvakis** [networkit/networkit#696] [Q&A]
Hello,

First of all, thanks for your great work! Secondly, I would like to ask you if you could provide the code based on which you generated the visualizations appearing on the paper:

> von Looz, Moritz, et al. Fast generation of dynamic complex networks with underlying hyperbolic geometry. KIT, Fakultät für Informatik, 2014.

Especially, I am interested in reproducing the two first figures also appearing in this [link](http://parco.iti.kit.edu/software/HRGs.shtml).

Thank you in advance!

**Comment by avdgrinten** [comment]
As far as I can tell, the figures were not directly generated with NetworKit - I assume that the graphs were exported and drawn using some other toolkit. Drawing the diagrams after the coordinates of the vertices are known should not be too complicated (e.g. using an arbitrary vector graphics library). @mlooz can you comment on this?
